applications/RomApplication/python_scripts/fluid_dynamics_analysis_rom.py

```diff
@@ -0,0 +1,46 @@
+from __future__ import absolute_import, division #makes KratosMultiphysics backward compatible with python 2.6 and 2.7
+
+import KratosMultiphysics
+import KratosMultiphysics.FluidDynamicsApplication
+import KratosMultiphysics.RomApplication as romapp
+from KratosMultiphysics.RomApplication import python_solvers_wrapper_rom as solver_wrapper
+from KratosMultiphysics.FluidDynamicsApplication.fluid_dynamics_analysis import FluidDynamicsAnalysis
+
+import json
+
+class FluidDynamicsAnalysisROM(FluidDynamicsAnalysis):
+
+    def __init__(self,model,project_parameters):
+        super(FluidDynamicsAnalysisROM,self).__init__(model,project_parameters)
```

Comment:
```suggestion
        super().__init__(model,project_parameters)
```

this is possible with python3 (we dropped python2 support :tada:)